CONTRIBUTING.rst

```diff
@@ -22,7 +22,7 @@ with your name and email::
     Signed-off-by: Jane Doe <developer@example.com>
```

Comment:
You will have to remove this signed-off too

Reply:
oops sorry, I guess this is the part of doc.

Reply:
yes

Reply:
no need to change.

Reply:
cool